### PR TITLE
Added h2h feature creation function

### PR DESCRIPTION
**Pull request by ivan-guo**

### CODE REVIEW

**Review by aryang13 — APPROVED** [review]

**Comment by ivan-guo** [comment]
@straw-hat-gupta Can you review this so I can merge? :)

File: h2h.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+    """
+    Creates the head to head features for each fighter in the dataset
+
+    Usage:
+        df = HeadToHead().create_h2h_feats(df)
+    """
+    def create_h2h_feats(self, df):
+        target_df = df
+
```
Comment (straw-hat-gupta) [inline h2h.py:16]: do we want ot change this to the copy method?

**Review by straw-hat-gupta — APPROVED** [review]

File: h2h.py (hunk 1 above)
Comment (straw-hat-gupta) [reply h2h.py:16]: other than that lgtm

Comment (ivan-guo) [reply h2h.py:16]: Ok bet I'll do that after I finish the test

**Pull request merged**